src/pages/editor/koni/components/EditorDetailPanel/MindDetailPanel.tsx

```diff
@@ -0,0 +1,19 @@
+import { CanvasPanel, DetailPanel, NodePanel } from 'gg-editor';
+
+import { Card } from 'antd';
+import React from 'react';
+import DetailForm from './DetailForm';
+import styles from './index.less';
+
+const MindDetailPanel = () => (
+  <DetailPanel className={styles.detailPanel}>
+    <NodePanel>
+      <DetailForm type="node" />
+    </NodePanel>
+    <CanvasPanel>
+      <Card type="inner" size="small" title="Canvas" bordered={false} />
+    </CanvasPanel>
+  </DetailPanel>
+);
+
+export default MindDetailPanel;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)